docs/src/tutorials/Nonlinear programs/space_shuttle_reentry_trajectory.jl

```diff
@@ -0,0 +1,331 @@
+#          R_e & = 20902900,                                    \qquad & c_2 & =  0.21286289 \times 10^{-3}, \\
+#            S & = 2690,                                        \qquad & c_3 & = -0.10117249 \times 10^{-5}.
+# \end{aligned}
+# ```
+
+# The reentry trajectory begins at an altitude where the aerodynamic forces are quite small
+# with a weight of $w = 203000$ (lb) and mass $m = w / g_0$ (slug), where $g_0 = 32.174$ (ft/sec$^2$).
+# The initial conditions are as follows:
+# 
+# ```math
+# \begin{aligned}
+#      h & = 260000 \text{ ft},  \qquad & v      & = 25600 \text{ ft/sec}, \\
+#   \phi & = 0      \text{ deg}, \qquad & \gamma & = -1    \text{ deg},    \\
+# \theta & = 0      \text{ deg}, \qquad & \psi   & = 90    \text{ deg}.
+# \end{aligned}
+# ```
+
+# The final point on the reentry trajectory occurs at the unknown (free) time $t_F$,
+# at the so-called terminal area energy management (TAEM) interface, which is defined by the conditions
+# 
+# ```math
+# h = 80000 \text{ ft}, \qquad v = 2500 \text{ ft/sec}, \qquad \gamma = -5 \text{ deg}.
+# ```
+
+# As explained in the book, our goal is to maximize the final crossrange,
+# which is equivalent to maximizing the final latitude of the vehicle,
+# i.e., $J = \theta(t_F)$.
+
+# ![Max crossrange shuttle reentry](https://i.imgur.com/1tUN3wM.png)
+
+# ### Approach 
+# We will use a discretized model of time, with a fixed number of discretized points, $n$.
+# The decision variables at each point are going to be the state of the vehicle and the controls commanded to it.
+# In addition, we will also make each time step size $\Delta t$ a decision variable;
+# that way, we can either fix the time step size easily,
+# or allow the solver to fine-tune the duration between each adjacent pair of points.
+# Finally, in order to approximate the derivatives of the problem dynamics, we will use either rectangular
+# or [trapezoidal](http://en.wikipedia.org/wiki/Trapezoidal_rule) integration.
+
+# > #### Disclaimer
+# > Do not try to actually land a Space Shuttle using this notebook! 😛
+# > There's no mesh refinement going on, which can lead to unrealistic trajectories having position
+# > and velocity errors with orders of magnitude $10^4$ ft and $10^2$ ft/sec, respectively.
+
+using JuMP
+import Interpolations
+import Ipopt
+
+## Global variables
+const w  = 203000.0  # weight (lb)
+const g₀ = 32.174    # acceleration (ft/sec^2)
+const m  = w / g₀    # mass (slug)
+
+## Aerodynamic and atmospheric forces on the vehicle
+const ρ₀ =  0.002378
+const hᵣ =  23800.0
+const Rₑ =  20902900.0
+const μ  =  0.14076539e17
+const S  =  2690.0
+const a₀ = -0.20704
+const a₁ =  0.029244
+const b₀ =  0.07854
+const b₁ = -0.61592e-2
+const b₂ =  0.621408e-3
+const c₀ =  1.0672181
+const c₁ = -0.19213774e-1
+const c₂ =  0.21286289e-3
+const c₃ = -0.10117249e-5
+
+## Initial conditions
+const hₛ = 2.6          # altitude (ft) / 1e5
```

Comment:
The subscripts aren't displaying correctly 

<img width="675" alt="image" src="https://user-images.githubusercontent.com/8177701/108784335-0e65c800-75d4-11eb-945c-8dc48a732409.png">

Reply:
Oh, that is weird. In which app are they not displaying correctly? On my system they look fine on the notebook, code, and docs in Chrome.

Reply:
![image](https://user-images.githubusercontent.com/4166093/108833714-2124d080-75c5-11eb-9dee-b6837a954fa6.png)

Reply:
On Mac, in both chrome and safari: 
<img width="513" alt="image" src="https://user-images.githubusercontent.com/8177701/108901516-d61bc380-767f-11eb-94e1-31a9887508ce.png">
I guess I don't have a font with `_s` installed.

Can we not just write out `h_s`? I assume I won't be the only one with this issue, so it is only going to cause confusion.

Looks good otherwise though!

Reply:
It seems you don't have the `\_s` or the `\_t`; I will replace those subscripts with plain `_s` and `_t`. It's weird because you do have others such as `\_r` and `\_e`. Did you notice any others not rendering?

Reply:
Unicode sub/superscripts are weird. Here is what displays for me: `ₐₑₕᵢⱼₖₗₘₙₒₚᵣₛₜᵤᵥₓ`.